entries/jQuery.isNumeric.xml

```diff
@@ -9,26 +9,31 @@
     </argument>
   </signature>
   <longdesc>
-    <p>The <code>$.isNumeric()</code> method checks whether its argument represents a numeric value. If so, it returns <code>true</code>. Otherwise it returns <code>false</code>. The argument can be of any type.</p>
+    <p>The <code>$.isNumeric()</code> method checks whether a value is a finite number, or is <code>Number</code>-castable to one. If so, it returns <code>true</code>. Otherwise it returns <code>false</code>. The argument can be of any type.</p>
```

Comment:
That's not accurate, though, because finiteness matters. What about "or would be cast to one by `Number`"?